packages/datatrak-web/src/utils/toast.ts

```diff
@@ -18,3 +18,14 @@ export const errorToast = (message: string) => {
     hideIconVariant: true,
   });
 };
+
+type Options = OptionsObject & { Icon?: OptionsObject['Icon']; hideCloseButton?: boolean };
```

Comment:
In light of brief in-person we had just now (Friday arvo)

```suggestion
interface Options extends OptionsObject {
  Icon?: OptionsObject['Icon'];
  hideCloseButton?: boolean
}
```